x-pack/plugins/security_solution/public/hosts/pages/hosts.test.tsx

```diff
@@ -35,6 +36,27 @@ jest.mock('../../common/components/search_bar', () => ({
 jest.mock('../../common/components/query_bar', () => ({
   QueryBar: () => null,
 }));
+jest.mock('../../common/components/visualization_actions', () => ({
+  VisualizationActions: jest.fn(() => (
+    <div data-test-subj="mock-viz-actions">{'mockVizAction'}</div>
+  )),
+}));
+
+jest.mock('../../common/lib/kibana', () => {
```

Comment:
I am almost sure the security solution already has a mock for useKibana and it has the cases plugin on it. Are you sure manually mocking cases here is necessary? check if  simply calling `jest.mock` in the file work.

Reply:
You can also use `x-pack/plugins/cases/public/mocks.ts`

Reply:
Yeah, need the manual mock to make the test work.

Reply:
You can always do

```
const casesMock = casesPluginMock.createStartContract();
casesMock.getRecentCases.mockImplementation(() => whatever);
```

It is up to you! I just wanted to mentioned in case you needed 🙂

Reply:
Definitely have to write way too much boiler plate for component jest tests as it is right now, probably a way where we could eliminate the need for that and speed up our tests at the same time.